libretro/libretro.cpp

```diff
@@ -579,13 +762,29 @@ bool retro_load_game(const struct retro_game_info *game)
       rom_loaded = Memory.LoadROM(game->path);
    else
    {
+      uint8 *biosrom = new uint8[0x100000];
+
       if (game->path != NULL)
       {
-         extract_basename(g_basename,   game->path, sizeof(g_basename));
+         extract_basename(g_basename, game->path, sizeof(g_basename));
          extract_directory(g_rom_dir, game->path, sizeof(g_rom_dir));
       }
 
-      rom_loaded = Memory.LoadROMMem((const uint8_t*)game->data ,game->size);
+      if (is_SufamiTurbo_Cart((uint8 *) game->data, game->size)) {
+         if (rom_loaded = LoadBIOS(biosrom,"STBIOS.bin",0x40000))
+            rom_loaded = Memory.LoadMultiCartMem((const uint8_t*)game->data, game->size, 0, 0, biosrom, 0x40000);
```

Comment:
I don't think this is actually necessary since the core can automatically load the bios, but we can take it in.